Gamma_Analysis.py

```diff
@@ -0,0 +1,362 @@
+"""
+This will perform a gamma analysis
+It will collect two spectra:
+M = Measurement Spectrum
+B = Background Spectrum
+"""
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import Gamma_Isotopes as ii
+import Gamma_Reference as ref
+import SPEFile
+import numpy as np
+import matplotlib.pyplot as plt
+import peakutils
+import pandas as pd
+import os
+
+EFFICIENCY_CAL_COEFFS = [-5.1164, 161.65, -3952.3, 30908]
+
+
+def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
+    """
+    Returns absolute efficiencies for a given set of energies, based on a
+    provided efficiency calibration. It takes an energy (in keV) and a set
+    of calibration coefficients.
+    The efficiency is calculated using the equation given below:
+    ln(efficiency) = c3*(E^3) + c2*(E^2) + c1*E + c0*E,
+    where E = ln(energy[keV])/energy[keV]
+    """
+    efficiency = []
+    for i in range(len(energy)):
+        efficiency.append(np.exp(coeffs[3] *
+                          (np.log(energy[i])/energy[i])**3 +
+                          coeffs[2]*(np.log(energy[i])/energy[i])**2 +
+                          coeffs[1]*(np.log(energy[i])/energy[i]) +
+                          coeffs[0]))
+    return efficiency
+
+
+def emission_rate(net_area, efficiency, livetime):
+    """
+    this function returns the emission rate of gammas per second
+    alongside its uncertainty.
+    """
+    emission_rate = [net_area[0]/(efficiency*livetime),
+                     net_area[1]/(efficiency*livetime)]
+    return emission_rate
+
+
+def isotope_activity(isotope, emission_rates, emission_uncertainty):
+    """
+    Isotope_Activity will determine the activity of a given radioactive isotope
+    based on the emission rates given and the isotope properties. It takes an
+    Isotope object and a given set of emission rates and outputs an activity
+    estimate alongside its uncertainty.
+    """
+    Branching_ratio = isotope.list_sig_g_b_r
```

Comment:
I realize how I've effectively read the previous comment for quite some time. My apologies for that. I also found that atom.io can do lower case for a big group of text, so future formatting will be much easier than the previous way I did it.